coalib/output/Logging.py

```diff
@@ -36,14 +58,23 @@ def configure_logging():
                     'INFO': 'blue',
                     'DEBUG': 'green'
                 }
+            },
+            'no-color': {
```

Comment:
Hm this handler is not used in this commit, you should move it to the commit where it's used first :+1: